### ISSUE TITLE
ClickableImageV2 or ClickableImageV3

### ISSUE THREAD

**Issue by fro0m** [qmlbook/qmlbook#23]
back-link: en/ch05/index.html#applying-animations

**Comment by jryannel** [comment]
Yes. It misses a short description what is the difference between V2 and V3. Is this what you meant?
